pom.xml

```diff
@@ -87,7 +87,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.integration.common</groupId>
       <artifactId>census-int-rate-limiter-client</artifactId>
-      <version>0.0.6</version>
+      <version>0.0.8-SNAPSHOT</version>
```

Comment:
Shouldn't this be an a proper version number? ie. not a snapshot

Reply:
It should be now, but when I first started working on this, you hadn't merged your code